addons-l10n/en/editor-delete-button.json

```diff
@@ -0,0 +1,7 @@
+{
+  "editor-delete-button/button-text": "Delete",
+  "editor-delete-button/modal-msg": "Are you sure you want to delete this project?",
+  "editor-delete-button/yes": "Yes",
+  "editor-delete-button/no": "No",
+  "editor-delete-button/modal-title": "Delete project"
```

Comment:
This should be capitalized to be consistent with Scratch's modals such as "Make a Block".